app/views/layouts/_navbar.html.haml

```diff
@@ -3,4 +3,5 @@
     %li= link_to "Vote!", new_ballot_path
     %li= link_to "Polls", polls_path
     %li= link_to "Beers", beers_path
-    %li= link_to "Add a beer", new_beer_path
+    -if can? :create, Beer
```

Comment:
Would you mind changing them to `- if`. That space would mean the world to me :smile: